docs/static/config-management.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[config-management]]
+== Configuration Management
+
+Logstash provides configuration management features to make it easier for you to
+manage updates to your configuration over time.
+
+[role="xpack"]
+[[logstash-centralized-pipeline-management]]
+=== Centralized Pipeline Management
+
+//REVIEWERS: I'm not sure if centralized pipeline management is the right term to use here, but if we want to have a generic container for lots of types of configuration management, we need to distinguish pipeline management.
```

Comment:
Yah, I was thinking that we'll probably want a different topic for each type of config management.